passing `int *' as argument 3 of `accept(int, sockaddr *, socklen_t *)' changes signedness

--- speech_tools/utils/EST_Server.cc.orig	Thu Nov 18 23:14:51 1999
+++ speech_tools/utils/EST_Server.cc	Sun Jun 11 17:27:17 2000
@@ -647,7 +647,7 @@
 
   while (connected() && 
 	 (csocket = accept(p_socket, (struct sockaddr *) &sin, 
-			   SOCKLEN_CAST &sin_size))>=0)
+			   (socklen_t *) &sin_size))>=0)
     {
       if (p_trace)
 	*p_trace << "connection " << csocket << "\n";
